module.js

```diff
@@ -0,0 +1,47 @@
+//import modules
+import Status from "./modules/status.js";
+import Tabs from "./modules/tabs.js";
+import Images from "./modules/images.js";
+import Footer from "./modules/footer.js";
+import Videos from "./modules/videos.js";
+import Channels from "./modules/channels.js";
+import Tags from "./modules/tags.js";
+import Layer1 from "./modules/layer1.js";
+import Layer2 from "./modules/layer2.js";
+
+//shotcut function
+function byId(id) {
```

Comment:
@DevBash1 It's difficult to understand what this function does. What does `byId` mean?

Try renaming this with a verb phrase. I want to be able to understand what it does by just looking at its name.

Reply:
Okay sir

Reply:
@devbash1 see this